src/style.css

```
@@ -1 +1,91 @@
 /* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, Times, serif;
```

You didnt add link to font. Add to html head:
`<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">`
```suggestion
  font-family: Roboto, sans-serif;
```